RELEASE.md

```diff
@@ -0,0 +1,32 @@
+**Design system - build and deploy HOWTO**
+
+To deploy a package to the npm you need to be logged in to an npm in your console. To confirm that you are logged in, you can use `npm whoami`. If you are not, use `npm login` and follow the instructions.
+
+To deploy you just have to do 'npm run deploy'. It will execute several commands you can run separately if you wish, namely:
+
+- 'check' - fire up linter and unit tests that are provided per package
+- 'prettier' - format the code across the packages
+- 'predeploy' - clean up old dist files and build new ones.
+- and finally deploy files to npm.
```

Comment:
I think we're missing the actual command to be executed here (`npm deploy`).

Reply:
It will repeat all commands above again - I am enumerating commands that can be executed before deploy.

Reply:
Ok, my bad. I've missed the text form 5th line